composer.json

```
@@ -14,7 +14,7 @@
     "keywords": ["Laravel", "Boxicons", "Blade UI Kit"],
     "require": {
         "php": "^7.2|^8.0",
```

Blade icons dropped PHP 7.3 support with v1.0
> Drop PHP 7.3
> - https://github.com/blade-ui-kit/blade-icons/blob/1.x/CHANGELOG.md#100-2021-03-21

```suggestion
        "php": "^7.4|^8.0",
```